src/FloorSubsystem.java

```diff
@@ -85,5 +89,10 @@ private void readDataFromFile() {
 			e.printStackTrace();
 		}
 	}
+	
+	public static void main(String[] args) {
```

Comment:
just wondering, why's there a main function in floor subsystem

Reply:
Forgot to take it out. I put it in to test. Taking it out now